.changeset/odd-rats-look.md

```diff
@@ -0,0 +1,5 @@
+---
+"@graphprotocol/grc-20": minor
+---
+
+Update `QUOTES_THAT_SUPPORT_CLAIMS_PROPERTY` and `ACADEMIC_FIELD_TYPE` values to be valid IDs
```

Comment:
We'll need to figure out the migration path for these since there's already data in the mainnet KG using these ids.

Reply:
What's KG?

One thing we could do is simply create new entities and change the entries manually. In this case in old version we still have old versions pointing to invalid IDs, but since the data is already on chain we can't do a simply SQL migration script or can we?